tests/integration/test_stores.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+
+
+@unittest.mock.patch("localstack.config.ALLOW_NONSTANDARD_REGIONS", True)
```

Comment:
@viren-nadkarni 

we usually use `monkeypatch.setattr(config, "ALLOW_NONSTANDARD_REGIONS", True)` here.
Any reason you're using unittest in this case?

Reply:
No particular reason at all, it just didn't cross my mind. I'll fix it when I retouch this next time